### PR TITLE
Js doc cleanup

### PR DESCRIPTION
**Pull request by rejas**
Lots of http: --> https: conversions (still some left to do though or replace with correct notes)
Cleaned up some tags

### CODE REVIEW

File: feature-detects/audio/webaudio.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "builderAliases": ["audio_webaudio_api"],
   "authors": ["Addy Osmani"],
   "notes": [{
-    "name": "W3 Specification",
+    "name": "W3 Spec",
```
Comment (patrickkettner) [inline feature-detects/audio/webaudio.js]: any particular reason for this?

File: feature-detects/css/flexbox.js [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "tags": ["css"],
   "notes": [{
     "name": "The _new_ flexbox",
-    "href": "http://dev.w3.org/csswg/css3-flexbox"
+    "href": "https://drafts.csswg.org/css-flexbox-1/"
```
Comment (patrickkettner) [inline feature-detects/css/flexbox.js]: @gregwhitworth while we are updating, would you consider the css drafts release to be the most correct one to link to in modernizr's docs?

File: feature-detects/elem/picture.js [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "authors": ["Scott Jehl", "Mat Marquis"],
   "notes": [{
     "name": "Specification",
-    "href": "http://picture.responsiveimages.org"
+    "href": "http://w3c.github.io/html/semantics-embedded-content.html#the-picture-element"
```
Comment (patrickkettner) [inline feature-detects/elem/picture.js]: 👎 on this change. The w3c html spec is dead, so we should not be linking to it for documentation. If you are against linking to the RIWG site as it is a redirect, then please update it to the site it is redirecting to

File: feature-detects/es5/strictmode.js [hunk 4]
```diff
@@ -2,10 +2,10 @@
 {
   "name": "ES5 Strict Mode",
   "property": "strictmode",
-  "caniuse": "sctrict-mode",
```
Comment (patrickkettner) [inline feature-detects/es5/strictmode.js]: lol

File: feature-detects/audio/webaudio.js (hunk 1 above)
Comment (rejas) [reply feature-detects/audio/webaudio.js]: was the only occasion where "Specification" was used, all other occurences were "W3 Spec" Just my little tidiness-spleen

File: feature-detects/elem/picture.js (hunk 3 above)
Comment (rejas) [reply feature-detects/elem/picture.js]: will change to the redirected page...

File: feature-detects/es5/strictmode.js (hunk 4 above)
Comment (rejas) [reply feature-detects/es5/strictmode.js]: by the way: I cant find a usage of the caniuse property, is that a TODO or where is it used?

**Pull request merged**